### ISSUE TITLE
Job launch duplication

### ISSUE THREAD

**Issue by vtjnash** [travis-ci/travis-ci#8616]
Travis appears to sometimes launch the same job multiple times in parallel on a single worker / job. For example, it started 9 copies in https://travis-ci.org/JuliaLang/julia/jobs/289647708. I've seen this problem a couple times before, but not very often.

**Comment by BanzaiMan** [comment]
Sorry about the issue. We are tracking this issue.

Also, a duplicate of #8429.
